### DISCUSSION TITLE
【报刊亭】来一份报纸吧！

### DISCUSSION THREAD

**Discussion by gaojiaobei** [share121/inter-knot#1417] [其他]
![微信图片_20240811171935](https://github.com/user-attachments/assets/9d5f6a1e-28a6-42e2-898a-e654c849a206)
欢迎在下方讨论哦！

**Comment by ZhuYuan-148** [comment]
最近捣毁了几个犯罪团伙，请市民们记得关注新闻。

**Comment by gaojiaobei** [comment]
治安官大人幸苦了！

**Comment by bin-lin2** [comment]
嗯呢！（治安官大人我是您的小邦布啊！）
嗯呢嗯呢！~~（治安官大人的宣传片我每天都有看上好几遍呢！）

**Reply by ZhuYuan-148** [reply]
诶？？？？
![Screenshot_20240810_151823](https://github.com/user-attachments/assets/44143f25-ffdc-4f57-a24b-f61b24598d2a)

**Comment by GanlinSama** [comment]
话说小卡格车这块儿是哪个部门在管？上街的时候总能看到犄角旮旯里有遗失的小卡格车
不是说重要的后勤物资么，就这么随地乱扔？

**Comment by Inter-Knot-User** [comment]
哪个邦布又偷了主人的手机来绳网追星？
